build_ngtf.py

```diff
@@ -173,39 +180,45 @@ def main():
     # flag is set to 0
     cxx_abi = "0"
 
-    if arguments.use_prebuilt_tensorflow:
-        print("Using existing TensorFlow")
-        command_executor(["pip", "install", "-U", "tensorflow==" + tf_version])
-
-        import tensorflow as tf
-        print('Version information:')
-        print('TensorFlow version: ', tf.__version__)
-        print('C Compiler version used in building TensorFlow: ',
-              tf.__compiler_version__)
-        cxx_abi = str(tf.__cxx11_abi_flag__)
+    if arguments.use_tensorflow_from_location != "":
+        print("Using TensorFlow from " + arguments.use_tensorflow_from_location)
```

Comment:
assert that use_tensorflow_from_location is a valid path

Reply:
Please add a comment on what is being done (if arguments.use_tensorflow_from_location != "":)

Reply:
Added comments here: https://github.com/tensorflow/ngraph-bridge/pull/107/files#diff-955b7e25a6094527d84683bcc9da8af9R73
and expanded help string of the option: https://github.com/tensorflow/ngraph-bridge/pull/107/files#diff-ece698560cf09ce24af8979586c76638R119

Reply:
Added some comments about the mechanics: https://github.com/tensorflow/ngraph-bridge/pull/107/commits/d0b7ae4b0aeb762dfd18529b8a9d0e70e05c47a2